generation/generation_test.go

```diff
@@ -88,7 +88,70 @@ workflows:
           requires:
             - test
 `,
-		}, {
+		},
+
+		{
+			testName: "cicd included in fallback config",
+			labels: labels.LabelSet{
+				labels.CICDGithubActions: labels.Label{
+					Key:   labels.CICDGithubActions,
+					Valid: true,
+					LabelData: labels.LabelData{
+						BasePath: ".",
+					},
+				},
+			},
+			expected: `# Couldn't automatically generate a config from your source code.
+# This is a generic template to serve as a base for your custom config
+# See: https://circleci.com/docs/configuration-reference
+# Stacks detected: cicd:github-actions:.
+version: 2.1
+jobs:
+  test:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - checkout
+      # Replace this with a real test runner invocation
+      - run:
+          name: Run tests
+          command: echo 'replace me with real tests!' && false
+  build:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - checkout
+      # Replace this with steps to build a package, or executable
+      - run:
+          name: Build an artifact
+          command: touch example.txt
+      - store_artifacts:
+          path: example.txt
+  deploy:
+    # This is an example deploy job, not actually used by the workflow
```

Comment:
This comment seems to not match the workflow definition below.

Reply:
Ideally this comment would only be present in the non-fallback case, where the job is commented out (IIRC)

Reply:
I removed this comment when it is a comment when generator returns a fallback config